stf.test/test/saros/stf/test/permissions/AllParticipantsFollowUserWithWriteAccessTest.java

```diff
@@ -35,6 +35,9 @@ public static void selectTester() throws Exception {
 
     Util.setUpSessionWithJavaProjectAndClass(
         Constants.PROJECT1, Constants.PKG1, Constants.CLS1, ALICE, BOB, CARL, DAVE);
+
+    // ensure all participants have successfully joined the session
+    Thread.sleep(5000);
```

Comment:
As written in the commit message, this delay is now needed as the test `followingUserOpenClassWhenFollowedUserOpenClass()` fails otherwise due to the timeout for the `waitUntilEditorOpen(...)` calls expiring before the session is completely initialized for all participants.

One guess I have is that this is due to the detection of when a session is completely initialized in the STF. With the previous logic, the session was first registered as partial sharing until all resources were synced, at which point the session would be upgraded to "completely shared" (aka the new normal).
With the new logic, the session is registered as shared (previously called "completely shared") from the start, meaning there is no longer a distinction for the setup stage.